playbooks/devel.yml

```diff
@@ -1,21 +1,28 @@
 ---
-- hosts: devel
+- hosts: all
   become: true
   vars:
     katello_repositories_version: nightly
-    foreman_installer_scenario: katello-devel
-    foreman_installer_options_internal_use_only:
-      - "--disable-system-checks"
-      - "--katello-devel-github-username {{ katello_devel_github_username }}"
-      - "--katello-devel-enable-ostree=true"
+    katello_repositories_environment: staging
+    foreman_installer_skip_installer: true
     foreman_installer_additional_packages:
-      - foreman-installer-katello-devel
+      - foreman-installer-katello
   roles:
     - etc_hosts
     - epel_repositories
     - ostree_repositories
     - puppet_repositories
     - foreman_repositories
     - katello_repositories
-    - foreman_devel
+    - foreman_installer
+
+- hosts: all
```

Comment:
Any reason to not do this in one block?

Reply:
Yup -- note how in the first block we need the installer role with the skip option to install the package and ensure all the modules and directories for the scenario are laid down. Then this block acts assuming that all exists.

Reply:
I assumed that since ansible is linear this wasn't needed.

Reply:
It is -- but I don't follow. The way I see it is:

 1) foreman-installer-katello needs to be installed (this is handled today by foreman_installer role)
 2) foreman_installer_devel_scenario lays down puppet-katello_devel, devel scenario, etc.
 3) foreman-installer needs to now run (this is handled by foreman_installer role)

Reply:
Oh yes, I missed that to install the scenario you need foreman-installer present since it relies on some directories that foreman-installer creates.